packages/store-sync/src/index.ts

```diff
@@ -7,4 +7,5 @@ export * from "./logToTable";
 export * from "./tablesWithRecordsToLogs";
 export * from "./tableToLog";
 export * from "./recordToLog";
-export * from "./logToRecord";
+
+export { logToRecord, type LogToRecordArgs } from "@latticexyz/store/internal";
```

Comment:
feels weird to export an internal thing but this let me move it to store

maybe we just remove this export since its intent is internal?